packages/cerebral/src/views/angularjs/module.js

```diff
@@ -0,0 +1,85 @@
+import angular from 'angular'
+import {Controller, provide} from '../../'
+import View from '../View'
+let controller = null
```

Comment:
do we need it here? I see only assignment on  row 75 and no consumptions anywhere